CompetitionTemplate/src/competition/subsystems/arm/ArmSubsystem.java

```diff
@@ -53,7 +60,11 @@ public boolean isArmAtMaximumHeight() {
     }
     
     public double getArmAngle() {
-        return encoder.getDistance();
+        return encoder.getDistance() * armEncoderDistancePerPulse.get() - armEncoderCalibrationHeight.get();
+    }
+    
+    public void calibrateArm() {
+        setArmMotorPower(armCalibrationPower.get());
```

Comment:
might vote for renaming this method more like setArmMotorToCalibratePower or something since this sounds like it should do more than it is.